README.md

```diff
@@ -271,27 +270,27 @@ But we don't just want to blindly do that. We're here to understand what's actua
 
 When we copied the things in the _Remote Repository_ into your _Dev Environment_ a few extra steps happened under the hood. 
 
 Git setup the _remote_ of your _Local Repository_ to be the _Remote Repository_ you cloned and gave it the default name `origin`. 
 
 >Your _Local Repository_ can track several _remotes_ and they can have different names, but we'll stick to the `origin` and nothing else for this tutorial. 
 
-Then it copied the two remote branches into your _Local Repository_ and finally it `checked out` _master_ for you. 
+Then it copied the two remote branches into your _Local Repository_ and finally it `checkout`ed _master_ for you. 
 
-When doing that another implicit step happens. When you `checkout` a branch name that has an exact match in the remote branches, you will get a new _local_ branch that is linked to the _remote_ branch. The _remote_ branch is the _upstream branch_ of your _local_ one. 
+When doing that, another implicit step happens. When you `checkout` a branch name that has an exact match in the remote branches, you will get a new _local_ branch that is linked to the _remote_ branch. The _remote_ branch is the _upstream branch_ of your _local_ one. 
 
-In the diagrams above you can see just the local branches you have. You can see that list of local branches by running `git branch`. 
+In the diagrams earlier you can see just the local branches you have. You can see that list of local branches by running `git branch`. 
 
 If you want to also see the _remote_ branches your _Local Repository_ knows, you can use `git branch -a` to list all of them.
 
 ![Remote and local branches`](img/branches.png)
 
 Now we can call the suggested `git push --set-upstream origin change_alice`, and `push` the changes on our branch to a new _remote_. This will create a `change_alice` branch on the _Remote Repository_ and set our _local_ `change_alice` to track that new branch. 
 
-> There is another option if we actually want our branch to track something that already exists on the _Remote Repository_. Maybe a colleague has already pushed some changes, while we were working on something related on our local branch, and we'd like to integrate the two. Then we could just set the _upstream_ for our `change_alice` branch to a new _remote_ by using `git branch --set-upstream-to=origin/change_alice` and from there on track the _remote_ branch.
+> There is another option if we actually want our branch to track something that already exists on the _Remote Repository_. Maybe a colleague has already pushed some changes, while we were working on something related to our local branch, and we'd like to integrate the two. Then we could just set the _upstream_ for our `change_alice` branch to a new _remote_ by using `git branch --set-upstream-to=origin/change_alice` and from there to track the _remote_ branch.
```

Comment:
The first one was actually meant as an _on_, i.e. _working on the branch_ 

Maybe instead replacing the `something related` with `some related issue` would make that sentence clearer?

Reply:
Yeah, so I misunderstood the sentence, good that we are rephrasing it. Your suggestion will work.